src/schemas/office-address-schema.ts

```diff
@@ -31,3 +31,30 @@ export const officeAddressSchema = {
     maxLength: maxLength(80)
   }
 }
+
+export const addressSchema = {
```

Comment:
What's the difference between this address schema and the one above? Consider giving it a more descriptive name and/or add a code comment explaining why we have 2 address schemas.

Reply:
Only difference is the validation on the Country and Region. I'm assuming the same rules don't apply to the offices as it does the directors being within Canada and BC. I believe we can use one schema and apply these conditionally. Will try!

Reply:
Renamed the `addressSchema` to `personAddressSchema` where applicable to differentiate between the Offices and Persons validation.

Reply:
* Future, perhaps we can refactor to one schema.